shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -161,6 +162,7 @@ async fn test_source_tls_and_cluster_tls() {
     collections::test(&connection).await;
     prepared_statements::test(&connection).await;
     batch_statements::test(&connection).await;
+    cluster::test(&connection).await;
```

Comment:
Can we not run the `test_topology_task` test as well here?

Reply:
It took a sizeable change so I did it in a separate PR https://github.com/shotover/shotover-proxy/pull/754